tcpdump.1.in

```diff
@@ -1145,6 +1145,14 @@ packet from the network and the time when an interrupt was delivered to
 the kernel to get it to read the packet and a delay between the time
 when the kernel serviced the `new packet' interrupt and the time when it
 applied a time stamp to the packet.
+.SS Interface
+.LP
+When multiple interfaces are selected the interface name is printed after
```

Comment:
"When multiple interfaces are selected"
should be
"When the `any` interface is selected on capture or when a `link-type LINUX_SLL2` capture file is read"
?

Reply:
I agree with that but imo it's wiser to just say "multiple interfaces". This way tcpdump can later add things like specifically selecting multiple interfaces (or other things I can't even think of) without risking deprecated documentation because nobody thought of this half sentence.

Reply:
`DLT_PFLOG` contains interface names too, but the output format is different, so it may be a good idea to say the new paragraph is specific to `SLL2`.